doc/user/content/sql/create-sink.md

```diff
@@ -193,47 +193,62 @@ If the topic does not exist, Materialize will use the Kafka Admin API to create
 
-When you create a sink, you must:
+    {{< note >}}
+For some deployment setups, you may need to persist the system catalog to stable storage across restarts. Without that metadata, new timestamps will be reassigned to existing offsets and data **will be republished** to the sink.
+   {{</ note >}}
 
-* Enable the `reuse_topic` option.
-* Optionally specify the name of a [consistency topic](#consistency-metadata) to store the information that Materialize will use to identify the last completed write. The names of the sink topic and the sink consistency topic must be unique across all sinks in the system. The name of the consistency topic may be provided via:
-    * The `CONSISTENCY TOPIC` parameter.
-    * The `consistency_topic` WITH option. **Note:** This option is only available to support backwards-compatibility. You will not be able to indicate `consistency_topic` and `CONSISTENCY TOPIC` or `CONSISTENCY FORMAT` simultaneously.
+In practice, each incoming event affects the final results exactly once, even if the stream is disrupted or Materialize is restarted. Because the implementation effectively relies on source events having replayable timestamps, only **Kafka sources** and materialized views defined on top of Kafka sources can be used in this context.
 
-  If not specified, a default consistency topic name will be created by appending `-consistency` to the output topic name.
+**Syntax**
 
-* If you are using a JSON-formatted sink, you must specify that the consistency topic format is AVRO. This is done through the `CONSISTENCY FORMAT` parameter. We're working on JSON support, but it's not available yet.
+```sql
+CREATE SINK quotes_sink
+FROM quotes
+INTO KAFKA BROKER 'localhost:9092' TOPIC 'quotes-eo-sink'
+CONSISTENCY (TOPIC 'quotes-eo-sink-consistency'
+             FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://localhost:8081')
+WITH (reuse_topic=true)
+FORMAT JSON;
+```
 
-Additionally, the sink consistency topic cannot be written to by any other process, including another Materialize instance or another sink.
-Key-based compaction is supported for the consistency topic and can be useful for controlling the topic's growth.
+To create a sink with exactly-once processing guarantees, you need to:
 
-Because this feature is still in beta, we strongly suggest that you start with test data, rather than with production. Please [escalate](https://github.com/MaterializeInc/materialize/issues/new/choose) any issues to us.
+* Set the `reuse_topic` option to `true`;
+* Optionally name the consistency topic. This name must be unique across all sinks in the Materialize instance. If not specified, a default name will be created by appending `-consistency` to the sink topic name.
+* Specify the [format](https://materialize.com/docs/sql/create-sink/#consistency_format_spec) of the consistency topic. Only Avro is supported. If you are using a JSON-formatted sink, you must use Avro as well (at least for now!).
```

Comment:
> If you are using a JSON-formatted sink, you must use Avro as well (at least for now!)

Couldn't find an issue for this one.

Reply:
I created https://github.com/MaterializeInc/materialize/issues/10163 for this